### DISCUSSION TITLE
CNCF TAG-Runtime discussion

### DISCUSSION THREAD

**Discussion by miao0miao** [pulzeai-oss/knn-router#9] [General]
Hello knn-router team,

I'm one of the co-chairs of the CNCF TAG-Runtime, I think it would be great for you to present/discuss the project in one of our meetings. For example, discuss things such as adoption, architecture, etc. You can also check CNCF TAG-network as this project seems to overlap between these two TAGs. 

Let me know if this something you'd be interested in doing. If yes, please feel free to add it to our [agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) 

Thanks!

**Comment by fabianbaier** [comment]
Thank you for reaching out and for your interest in the knn-router project. We are honored by the invitation to present and discuss our project at an upcoming CNCF TAG-Runtime meeting. The opportunity to delve into the adoption and architecture of knn-router, and to explore its intersection with CNCF TAG-Network, sounds fantastic.

We are very interested in participating. Could you please share more details about the format you envision for our presentation and any specific areas of interest the TAG members might have? This will help us prepare and tailor our content effectively.

Additionally, could you guide us on how to add our discussion to the agenda? We want to ensure we follow the proper procedure.

Thank you once again, and I look forward to your guidance and the possibility of collaborating closely with the CNCF community.

Feel free to ping me directly or join our slack https://pulzeaicommunity.slack.com

Best regards,

Fabian Baier  
CEO Pulze.ai Inc.
knn-router Team
